e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -555,6 +555,36 @@ test.describe('Example Imagery in Time Strip', () => {
     // Navigate to timestrip
     await page.goto(timeStripObject.url);
   });
+
+  test.describe('Example Imagery in Flexible layout', () => {
+    let flexibleLayout;
+    test.beforeEach(async ({ page }) => {
+      await page.goto('./', { waitUntil: 'domcontentloaded' });
+
+      flexibleLayout = await createDomainObjectWithDefaults(page, { type: 'Flexible Layout' });
+
+      // Create Example Imagery inside the Flexible Layout
+      await createDomainObjectWithDefaults(page, {
+        type: 'Example Imagery',
+        parent: flexibleLayout.uuid
+      });
+
+      // Navigate back to Flexible Layout
+      await page.goto(flexibleLayout.url);
+    });
+
+    test('Can double-click on the image to view large image', async ({ page }) => {
+      // Double-click on the image to open large view
+      await page.locator('.image-wrapper').dblclick();
```

Comment:
Same thing here. Is there a way we could avoid `.locator()` ?

Reply:
Sure! But I'm not sure if I did it right, would `waitForSelector('.image-wrapper')` work? If not, what would you recommend me to do (or use as the method to get the element)?

Reply:
We're moving away from `page.locator()` in general and we now prefer to use ARIA based methods such as `page.getByRole()`. One way we could improve this could be by adding an accessible name (such as an `aria-label` to [the img element](https://github.com/360macky/openmct/blob/a52482e624ad42bf61f97ac79e3af5eff10463cc/src/plugins/imagery/components/ImageryView.vue#L69). We could then update the locator to something like this:

```js
await page.getByRole('img', { name: 'Focused Image' });
```

Reply:
Thank you! I have replaced the `pageSelector` of the image wrapper and the background image with `getByRole`.